frontend/src/pages/CompareV2.tsx

```diff
@@ -226,14 +350,18 @@ function CompareV2(props: PageProps) {
     staleTime: Infinity,
   });
 
-  // TODO(zpChris): The pending work item of displaying visualizations will need these filters.
-  if (artifactTypes) {
-    filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.SCALAR_METRICS);
-    filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.CONFUSION_MATRIX);
-    filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.ROC_CURVE);
-    filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.HTML);
-    filterRunArtifactsByType(runArtifacts, artifactTypes, MetricsType.MARKDOWN);
-  }
+  useEffect(() => {
+    if (runs && mlmdPackages && artifactTypes) {
+      const runArtifacts: RunArtifacts[] = getRunArtifacts(runs, mlmdPackages);
```

Comment:
Does the `mlmdPackages: MlmdPackage[]` object always have the items in the same order? I don't know the backend implementation, I want to make sure that I don't need to sort those items on the frontend to make sure they display in the same order every time.

Reply:
`MlmdPackage[]` is sorted based on runIds. And executions/artifacts/etc. within each MlmdPackage is sorted by executionID/artifactID/etc.

Reply:
Ok perfect, that works great - thank you!